src/templates/customer/nr_view_order/order.template.html

```diff
@@ -1,39 +1,27 @@
-<!--#Start Column Left# ##-->
-<!--##[%load_template file:"column_left/template.html"%][%END load_template%]##-->
-<!--# End Column Left# ##-->
-
-
-<!--# Start Breadcrumb# ##-->
-<ul class="breadcrumb" aria-label="Breadcrumb"> 
-<li><a href="[@config:home_url@]">Home</a></li>
-<li><span class="divider">/</span> <a href="[%URL page:'account'%][%END URL%]">My Account</a></li>
-</ul>
-<!--# End Breadcrumb# ##-->
-<div class="page-header">
-<h1>View Order / Invoice</h1>
+<div class="col-xs-12">
+	<ul class="breadcrumb" aria-label="Breadcrumb"> 
+		<li><a href="[@config:home_url@]">Home</a></li>
+		<li><a href="[%url page:'account'/%]">My Account</a></li>
+	</ul>
+	<div class="page-header">
+		<h1>View Order / Invoice</h1>
+	</div>
+	<script language="javascript" type="text/javascript">
+		function print_invoice() {
+			var win = window.open('[%url page:"account" type:"print" fn:"invoice" id:"[@order_id@]"/%]', 'print_invoice', 'width=740,height=400,scrollbars=yes,resizeable=yes');
+		}
+	</script>
+	<div class="row">
+		<div class="col-xs-12">
+			<input type="button" class="btn btn-default pull-left" value="Print Order / Invoice" onClick="print_invoice()">
+			[%show_order id:'[@order_id@]'%]
+				[%param *header%]
+					[%if [@amount_owed@] > 0%]
+						<button class="btn btn-success pull-right" onclick="location.href='[%url page:'account' type:'pay_order' qs:'action=paynow' id:'[@order_id@]'/%]'"><i class="fa fa-check icon-white"></i> Pay Now</button>
+					[%/if%]
+				[%/param%]
+			[%/show_order%]
+		</div>
+	</div>
+	[%print_order id:'[@order_id@]'/%]
 </div>
-
-<script language="javascript" type="text/javascript">
-function print_invoice() {
-	var win = window.open('[%URL page:"account" type:"nr_print" fn:"invoice" id:"[@order_id@]" qs:"code=[@code@]"%][%END URL%]', 'print_invoice', 'width=740,height=400,scrollbars=yes,resizeable=yes');
```

Comment:
Might need to check if `qs:"code=[@code@]"` is important.

It's not in the nr_view_order/quote template though...